lib/graphql/queries.ts

```diff
@@ -0,0 +1,133 @@
+export const GET_USER = `query {
```

Comment:
I like the idea to outsource all graphql queries, but can we discuss if it makes sense to keep them (as constants) in their corresponding files instead of one giant `queries.ts`?

Reply:
you're right, I'll move it back them back to corresponding files